### PR TITLE
phoebe #24: Updated session.ts under api and player.ts under database

### PR DESCRIPTION
**Pull request by tsaichenlo**
#24 Updated getParticipants to parse sessionId correctly. 
#24 Wrote the getPlayersForSession function

### CODE REVIEW

**Comment by leon332157** [comment]
#24

File: src/backend/api/session.ts [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+    const sessionId: number = parseInt(req.query.sessionId as string);
+    const players: string[] | undefined = await playerDatabase.getPlayersForSession(sessionId);
+
+    if (players) {
+        res.json({ players });
+    }
+    else {
+        res.status(404).send(`Game not found with id: ${req.query.sessionId}`);
+    }
     res.json(["kevin", "selena", "jiejie", "xiaocong"]);
```
Comment (leon332157) [inline src/backend/api/session.ts:17]: remove dummy data

File: src/backend/database/player.ts [hunk 2]
```diff
@@ -94,6 +94,21 @@ export class PlayerDatabase {
...
 
+  async getPlayersForSession(sessionId: any): Promise<string[] | undefined> {
+    try {
+      const players = await PlayerModel.findAll({
+        attributes: ['name'],
+        where: { sessionId },
+      });
+      return players.map(player => player.name);
+    } catch (error) {
+      (reason: any) => {
```
Comment (leon332157) [inline src/backend/database/player.ts:105]: should be 
```js
catch (e) {
    console.error(e)
}
```

**Review by leon332157 — CHANGES_REQUESTED** [review]
small fix

**Comment by eungyuShim** [comment]
looks good

**Review by leon332157 — APPROVED** [review]
good now

**Pull request merged**